Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'safemode', '~> 1.2'
 gem 'fast_gettext', '~> 0.8'
 gem 'gettext_i18n_rails', '~> 1.0'
 gem 'i18n', '~> 0.6.4'
+gem 'rails-i18n', '~> 3.0.0'
```

Comment:
The only other gem I found using this scheme is audited-activerecord, and nobody is promising us they will keep the same, which might lead to head-scratching-type-breakage.